cognee-frontend/src/ui/Partials/SearchView/SearchView.tsx

```
@@ -108,10 +138,32 @@
+function convertToSearchTypeOutput(systemMessages: [], searchType: string): string {
+  switch (searchType) {
+    case 'INSIGHTS':
+      return systemMessages.map((message: InsightMessage) => {
+        const [node1, relationship, node2] = message;
+        if (node1.name && node2.name) {
+          return `${node1.name} ${relationship.relationship_name} ${node2.name}.`;
+        }
+        return '';
+      }).join('\n');
+    case 'SUMMARIES':
+      return systemMessages.map((message: { text: string }) => message.text).join('\n');;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)